### PR TITLE
Canned Bosh Director for Acceptance Tests

### PR DESCRIPTION
**Pull request by peanball**
Add script for creating a running bosh director in Docker that can be resurrected.

This is for running locally only so far.
Integration with the concourse pipeline will be done in a future activity.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -91,6 +91,7 @@ GEM
     unicode-display_width (2.2.0)
 
 PLATFORMS
+  x86_64-darwin-21
```
Comment (peanball) [inline Gemfile.lock:94]:
```suggestion
```

File: acceptance-tests/run-local.sh [hunk 2]
```diff
@@ -35,7 +35,7 @@ fi
 
 # Build acceptance test image
 pushd "$SCRIPT_DIR/../ci" || exit 1
- docker build -t haproxy-boshrelease-testflight .
+ docker build --no-cache -t haproxy-boshrelease-testflight .
```
Comment (peanball) [inline acceptance-tests/run-local.sh:38]: why no-cache?

File: acceptance-tests/run-local.sh [hunk 3]
```diff
@@ -5,13 +5,13 @@ set -e
...
-    git status
-    exit 1
-fi
+# if [ "$(git status -s | wc -l)" -gt 0 ]; then
+#     echo "You have changes in your Git repository. Commit or clean (e.g. git clean -f) before running."
+#     echo "The build will fail otherwise."
+#     echo "Git Status:"
+#     git status
+#     exit 1
+# fi
```
Comment (peanball) [inline acceptance-tests/run-local.sh:14]:
```suggestion
```
This has now been replaced by calling `bosh create-release --force`, which will take local changes, even if uncommitted, into account.

File: ci/scripts/acceptance-tests [hunk 4]
```diff
@@ -18,39 +18,43 @@ cd ${REPO_ROOT:?required}
...
-source /tmp/local-bosh/director/env
-
 echo "----- Creating candidate BOSH release..."
 bosh -n reset-release # in case dev_releases/ is in repo accidentally
 
-bosh create-release
+time bosh create-release --force
+
+echo "uploading release. Vars:"
+export
```
Comment (peanball) [inline ci/scripts/acceptance-tests:42]:
```suggestion
echo "uploading release. Vars:"
env
```

**Comment by maxmoehl** [comment]
We decided to not pursue this approach further since it proved to be difficult to get working and is likely to break in the future.

**Pull request closed without merging**